flocker/acceptance/datahttp.py

```diff
@@ -0,0 +1,49 @@
+"""
+HTTP server that writes data to a specified file on POST, or reads and
+returns data from a specified file on GET.
+"""
+
+from urlparse import urlparse
+try:
+    from urlparse import parse_qs
+except ImportError:
+    from cgi import parse_qs
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_POST(s):
+        length = int(s.headers['content-length'])
```

Comment:
If you want to make this flexible, so that you can write to different locations, I suggest making the path a command-line argument.